Fixing the pull request link

HeyItsGwen opened the pull request:
Making a pull request, to get a pull request link. Didn't read ahead and merged into master.

HeyItsGwen committed:
Fixing the pull request link

Merged to master, before reading the instructions not to

Redvanisation reviewed (changes requested):
### Hello @HeyItsGwen and @aracelicaes  🌟

### **Good work on this project, but we need to fix the issues below before moving on ⬇️** 

**- First of all, it is a collaborative project, you guys need to have a reasonable number of commits made by both of you and not only one commit.**

**- The commits need to have descriptive names on the changes you are pushing to the repo.**

**- Please write a brief description of the changes you made on the next pull request.** 

### Please make the following changes:

1. The video needs some margin-left to position it just like on the original website.

2. Align the elements under the main video make them look like the original website.

3. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input).

4. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

HeyItsGwen committed:
Make video responsive

Changed the width to a percentage with a max width to stop oversizing.

HeyItsGwen committed:
Cosmetic changes

Changed the location of the share button to more match the example, added margins and centered the views, like, and dislike button to match more, and added a bottom margin to the title to make it look nicer.

HeyItsGwen committed:
re-commit cosmetic changes

some margins didn't commit from the last commit for some reason: added margin under title and avatar, and add width and alignment to views/like/dislike.

HeyItsGwen committed:
More cosmetic changes

Made background colors of page/sections more closely match the example, added more recommended videos and changed the max/min dimensions of the thumbnails to make it look nicer on browser resize.

aracelicaes committed:
"aside" added padding

Added padding to the asside section so the videos aren't on the very edge.

aracelicaes committed:
Added detail to "sign in button"

Added background color, font color, border and padding to sign in button.

aracelicaes committed:
added alt

Added the missing alt to the images and icons located in the top nav bar inside the <li> elements

aracelicaes committed:
added lang

fixed error on html, added lang=en-us

aracelicaes committed:
added metas

added meta charset
added meta viewport

aracelicaes committed:
added heading to video

Added heading to video, also added CSS to make it invisible.

HeyItsGwen committed:
Updated the README

Made readme more fleshed out and explanatory to fix what was needed for our code review.

HeyItsGwen committed:
Merge branch 'master' into NavBar-Creation

aracelicaes committed:
Fixing github issues

Fixed repeated elements that loaded with recloning.

HeyItsGwen commented:
Fixed issues from first code review, resubmitting for the second code review

imhta reviewed (approved):
# Approved ✔️ 
## Hi @HeyItsGwen  and @aracelicaes 👋  
### **Congratulations 🎉 , You have done good job.**
<img src="https://media.tenor.com/images/b16c58937bc0195cadabf759e60ff5ad/tenor.gif">

### You can merge this project and move to the next project.
>#### Don't forget to submit the project completion form
😄Happy coding, 
**Pon Muthu Selvam N**

**[LinkedIn](https://www.linkedin.com/in/ponmuthuselvam)** 
**[Github](https://github.com/imhta)**
